src/event/special_user_message.rs

```diff
@@ -345,7 +359,7 @@ fn deploy_command<'a>(
             "execute": true
         });
         let client = reqwest::Client::new();
-        client
+        let res = client
```

Comment:
this code threw me for a loop because the `res` variable only exists to make the `res_json` variable (right?).
I would've just renamed this `let res_json = ` and chained the `.json()` call onto the end instead of having two statements where the second just picks up where the first left off.

Reply:
I think I did this to avoid the borrow checker, which could be baseless worry

Reply:
ah yeah, depending on if `.json` takes ownership or not that could cause some issues.

Reply:
still, what's the point of having such a powerful compiler if you're not going to give it a chance to be useful? heh